modules/helm-installing-a-helm-chart-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="installing-a-helm-chart-on-an-openshift-cluster.adoc_{context}"]
+
+= Installing a Helm chart on an OpenShift cluster
+
+.Prerequisites:
```

Comment:
These Prereq and Procedure headings shouldn't have a colon after them.

Throughout the PR